ModuleNotFoundError: No module named 'aioredis.client' when running sanic tests

```
  __________________ ERROR collecting tests/sanic/test_sanic.py __________________
  ImportError while importing test module '/home/runner/work/python-dockerflow/python-dockerflow/tests/sanic/test_sanic.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  /opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  tests/sanic/test_sanic.py:11: in <module>
      import sanic_redis.core
  .tox/py39-s20/lib/python3.9/site-packages/sanic_redis/__init__.py:1: in <module>
      from .core import SanicRedis
  .tox/py39-s20/lib/python3.9/site-packages/sanic_redis/core.py:1: in <module>
      import aioredis.client
  E   ModuleNotFoundError: No module named 'aioredis.client'
```

Seen in https://github.com/mozilla-services/python-dockerflow/runs/4717358686?check_suite_focus=true for instance, which contains completely unrelated changes.

This is because `sanic-redis` 0.3.0 was released, depending on aioredis 2.x, which we have restricted to 1.x. Medium to long term we should probably change this to support aioredis 2.x, but in the short term, to unblock the situation, restrict `sanis-redis` to 0.2.x in tests.

--- a/tests/requirements/sanic.txt
+++ b/tests/requirements/sanic.txt
@@ -3,5 +3,5 @@
 aiohttp
 aioredis<2.0.0
 Sanic
-sanic_redis
+sanic_redis<0.3.0
 uvloop>=0.14.0rc1

